frontend/app/api/auth/[...nextauth]/route.ts

```
@@ -0,0 +1,76 @@
+      return token;
+    },
+  },
+  pages: {
+    signIn: "/login",
+  },
+  session: {
+    strategy: "jwt",
+  },
+};
+
+const handler = NextAuth(authOptions) as any;
```

ESLint

[error] Do not use any type assertions.

(@typescript-eslint/consistent-type-assertions)